epub33/rs/index.html

```diff
@@ -1170,24 +1146,124 @@ <h2>Open Container Format Processing</h2>
 				<h3>OCF Abstract Container</h3>
 
 				<section id="sec-container-iri">
-					<h4>Relative URLs for Referencing Other Components</h4>
+					<h4>URL of the Root Directory</h4>
 
-					<p>For <a href="https://url.spec.whatwg.org/#relative-url-with-fragment-string"
-							>relative-URL-with-fragment strings</a> [[URL]], Reading Systems MUST determine the <a
-							href="https://url.spec.whatwg.org/#concept-base-url">base URL</a> [[URL]] according to the
-						relevant language specifications for the given file formats. For example, CSS defines how
-						relative URL references work in the context of CSS style sheets and property declarations
-						[[CSSSnapshot]].</p>
+					<p>Reading Systems MUST assign a URL [[URL]] to the <a>Root Directory</a> of the <a>OCF Abstract Container</a>. This URL is called the <a data-cite="epub-33#dfn-container-root-url">container root URL</a>. The URL itself is implementation specific, but:</p>
+
+					<ul>
+						<li>The result of <a data-cite="url#concept-url-parser">parsing </a> [[URL]] "<code>/</code>" with the <a>container root URL</a> as <a data-cite="url#concept-base-url">base</a> MUST be the <a>container root URL</a>.</li>
```

Comment:
Same comments as in the other doc for the first two items.

Reply:
See https://github.com/w3c/epub-specs/pull/1898#discussion_r747448586

Reply:
I have carried over the changes, ie, https://github.com/w3c/epub-specs/pull/1898#discussion_r748038719 applies here, too. Just to clean up, I will resolve this conversation, it is "only" a consequence of the other one after all.